vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_console_01_consoles.crd.yaml

```diff
@@ -77,6 +77,49 @@ spec:
                     - Azure
                     - ROSA
                     type: string
+                  capabilities:
+                    description: capabilities defines an array of capabilities that
+                      can be interacted with in the console UI. Each capability defines
+                      a visual state, by which it will get rendered. Available capabilities
+                      are LightspeedButton. Each of the available capabilities may
+                      appear only once in the list.
+                    items:
+                      description: Capabilities contains set of UI capabilities and
+                        their state in the console UI.
+                      properties:
+                        name:
+                          description: name is the unique name of a capability. Available
```

Comment:
```suggestion
                          description: Unique name of a capability. Available
```

Reply:
This is a description for a GOlang's structure field. This description should always start with the lowercased name of the field. 
Check https://github.com/openshift/api/pull/1956/files where this change originates.